service/discovery/k8s/compute.go

```diff
@@ -70,14 +70,23 @@ func (k k8sComputeDiscovery) List() ([]voc.IsResource, error) {
 }
 
 func (k k8sComputeDiscovery) handlePod(pod *v1.Pod) voc.IsCompute {
-	return &voc.ContainerResource{
+	r := &voc.ContainerResource{
 		ComputeResource: voc.ComputeResource{
 			Resource: voc.Resource{
-				ID:           fmt.Sprintf("/namespaces/%s/containers/%s", pod.Namespace, pod.Name),
+				ID:           voc.ResourceID(getContainerResourceID(pod)),
```

Comment:
The ContainerResource is filled with pods, shouldn´t we fill it with containers instead?